packages/create/README.md

```diff
@@ -148,6 +149,15 @@ Calling `sap-ux change inbound` allows replacing the Inbound FLP configurations
 sap-ux change inbound [path]
 ```
 
+## sap-ux convert
+Calling `sap-ux convert` allows converting an app to a new feature.
```

Comment:
```suggestion
Executing `sap-ux convert` converts an app to a new feature.
```

Reply:
Done in fe3e4ffa7a57333062a7629ce48b4f040708751f